protocol/messenger.go

```diff
@@ -3800,7 +3815,7 @@ func (m *Messenger) encodeChatEntity(chat *Chat, message common.ChatEntity) ([]b
 		if err != nil {
 			return nil, err
 		}
-	case ChatTypePublic, ChatTypeProfile:
+	case ChatTypePublic, ChatTypeProfile, ChatTypeTimeline:
```

Comment:
Same here, do you ever save messages for a timeline chat?

Reply:
no thanks